README.md

```diff
@@ -49,7 +49,7 @@ To have a website that caters all needs that a CSE UG has.
 ##### Installation :
 
 * create a virtual environment and activate it (google it)
-* Install dependencies into it ```pip3 insall -r requirements.txt```.
+* Install dependencies into it ```pip3 install -r requirements.txt``` or ```python -m pip install -r requirements.txt```.
```

Comment:
* use python3, that is the reason you removed f' ' string, Please use f' ' string and python3